src/lib.rs

```diff
@@ -1487,21 +1487,14 @@ impl Build {
             ToolFamily::Msvc { .. } => {
                 cmd.push_cc_arg("-nologo".into());
 
-                let crt_flag = match self.static_crt {
-                    Some(true) => "-MT",
-                    Some(false) => "-MD",
-                    None => {
-                        let features = self
-                            .getenv("CARGO_CFG_TARGET_FEATURE")
-                            .unwrap_or(String::new());
-                        if features.contains("crt-static") {
-                            "-MT"
-                        } else {
-                            "-MD"
-                        }
-                    }
-                };
-                cmd.push_cc_arg(crt_flag.into());
+                // Since this crate doesn't actually make decisions about
+                // final application linking, it's only appropriate to not
+                // interfere with the process. This can be achieved with
+                // -MT -Zl, because object modules compiled with these flags
+                // can be linked with either VCCRT version, static or dynamic,
+                // debug or release...
```

Comment:
Well, was it actually non-trivial to find the -Zl documentation? It's available even in `cl /?` output... I'd argue that **if** something needs to be added, then it would be more appropriate to discuss the subtlety of the -MT in the context. The symbol decoration part... Either way, I'd wait till maintainers make the choice which direction will be preferred, this, #717 or #725 ;-)